modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+[id="enabling-assisted-installer-service-on-bare-metal_{context}"]
+
+.Enabling assisted installer service on bare metal
```

Comment:
```suggestion
Enabling assisted installer service on bare metal
```
This needs to be a heading, not a title.

Reply:
Updated as suggested.